app/views/layouts/sinatra/_navigation.html.erb

```diff
@@ -0,0 +1,27 @@
+<nav class="navbar ">
+  <div class="navbar-brand">
+    <a class="navbar-item" href=<%= uri('') %>>
+      Contrast Security Ruby - Sinatra
+    </a>
+  </div>
+</nav>
+<div class="columns">
+  <aside class="column is-2 hero aside is-fullheight" style="background: lightgray">
+    <div>
+      <div class="main">
+        <ul class="menu-list">
+          <li><a href=<%= uri('/..') %>>Contrast Security Ruby - Rails</a></li>
```

Comment:
Pretty sure this should say "Sinatra" instead of Rails.

Reply:
https://github.com/Contrast-Security-OSS/vulneruby_engine/pull/17/files#diff-644426cc03ff98f42e016f6a83aed508R4 is the title of the page

This is the link back to the rails application